### ISSUE TITLE
Noisy voice call on 4.2.0.21

### ISSUE THREAD

**Issue by xakod** [Michal-Szczepaniak/Yottagram#23]
Hey, i tried you client, it looks very nice but i cannot use voice calls. My interlocutor cannot hear me, he hears only part of my words and a lot of noise. I tried incomming and outcomming calls to different devices, looks like problem is on the sfos side.
The problem exists on 4.2.0.21 and 4.3.0.15 with yottagram-0.2.9

**Comment by Michal-Szczepaniak** [comment]
try to go to options and adjust voice call settings?

**Comment by xakod** [comment]
> try to go to options and adjust voice call settings? 

Yep, it doesnt help at all. Only echo cancellation gives something.
